src/collectd_dogstatsd.py

```diff
@@ -106,13 +106,12 @@ def dims_from_tags(tags, metric):
         return ret
     for tag in tags:
         parts = tag.split(":", 1)
-        if len(parts) == 0:
-            # Skip labels
-            continue
-        if len(parts) == 1:
-            collectd.error(
-                "skipping malformed dogstatsd tag: '{0}' for metric: '{1}'"
+        # Skip labels and malformed tags
+        if len(parts) < 2:
+            collectd.warn(
+                "dropping malformed dogstatsd tag: '{0}' for metric: '{1}'"
                 .format(tag, metric))
+            collectd.notice("dogstatsd labels are not supported")
```

Comment:
Is there any difference between a label and malformed tag as far as our plugin can tell?

Reply:
tags are key/value and labels are only a single value with no key.  We can't tell the difference when they come in.  So i warn that a malformed tag came through and note that labels are unsupported.